lib/report/generateReport.ts

```diff
@@ -201,18 +211,77 @@ export class CoverageView {
           runtimeExceptions: element.runtimeExceptions,
         });
       });
+
+      const generalErrorsInnerOrigin = this.validationResultsForRendering.filter((x) => {
```

Comment:
Nit: `generalErrorsInnerOrigin` means "any validation result that has errors." Why not call this `failingValidationResults`? 

Effectively:

- `validationResultsForRendering` (also a long name but don't change this IMO)
- `errorValidationResults`
- `passValidationResults`
- Where error and pass added together equals `validationResultsForRendering`.